src/harden.yml

```diff
@@ -0,0 +1,10 @@
+---
+- hosts: all
+  name: Hardening tasks
+  become: yes
+  become_method: sudo
+  roles:
+    - role: harden
+      # vars:
+      #   sshd_admin_net:
+      #     - 127.0.0.1/32
```

Comment:
@jsf9k - Do we want to remove these commented-out lines?

Reply:
I confirmed with @jsf9k via chat that these lines are fine to be removed, so I did so in https://github.com/cisagov/openvpn-packer/pull/27/commits/c382b7703f2a5a7d333191e4e36953d0d95d223a.